bundles/net.tourbook.cloud/src/net/tourbook/cloud/strava/messages.properties

```diff
@@ -10,16 +10,9 @@ Log_UploadToursToStrava_002_NoTourTitle  = {0} -> Tour title is missing
 Log_UploadToursToStrava_003_ActivityLink = {0} -> Uploaded Activity Link: <br><a href="https://www.strava.com/activities/{1}">https://www.strava.com/activities/{1}</a></br>
 Log_UploadToursToStrava_003_UploadStatus = {0} -> Upload Id: "{1}". Creation Activity Status: "{2}"
 Log_UploadToursToStrava_004_UploadError  = {0} -> Error while uploading the tour: "{1}"
-Log_UploadToursToStrava_005_End          = Performed in %.3f s
 
-PrefPage_Account_Information_AccessToken_Label    = Access token
 PrefPage_Account_Information_AthleteName_Label    = Athlete's name
 PrefPage_Account_Information_AthleteWebPage_Label = Athlete's web page
-PrefPage_Account_Information_ExpiresAt_Label      = Expires at
-PrefPage_Account_Information_RefreshToken_Label   = Refresh token
```

Comment:
Fixed
![image](https://user-images.githubusercontent.com/25235665/107134020-87aeac80-68ab-11eb-8996-d42787d35f25.png)

Reply:
This, morning, laying in bed, this thought come in my mind: Currently there are 4 different message files multiplied by language, and all are very small. Should they be merged into one ?

When you run the externalize string tool, it creates a new message file for each package. It would be a horror for me to support this structure as there would be thousands more files.

![mt-cloud-translations](https://user-images.githubusercontent.com/1283445/107139761-f3316200-691d-11eb-8522-c1e377d0ae0f.png)

Reply:
I have put all the translation files in 1 location only : net.tourbook.cloud

![image](https://user-images.githubusercontent.com/25235665/107170332-4ab2ea80-697d-11eb-9000-b9f0ec353e08.png)